tests/bmp/test_frr_bmp_sanity.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+from tests.common.helpers.bgp import run_bgp_facts
+from bmp.helper import enable_bmp_feature, disable_bmp_feature
+
+pytestmark = [
+    pytest.mark.topology('any', 't0-sonic', 't1-multi-asic'),
+    pytest.mark.device_type('vs')
+]
+
+
+def test_frr_bmp_bgp_facts(duthosts, enum_frontend_dut_hostname, enum_asic_index):
```

Comment:
>test_frr_bmp_bgp_facts [](http://example.com/codeflow?start=4&length=22)

The purpose of this testcase is to prevent regression. Did you test it with a bad image, and see the result is failure? I guess it will catch monit ERR log.